pkg/snapshot/restorer/restorer_suite_test.go

```diff
@@ -105,18 +105,19 @@ func cleanUp() {
 
 	//for the negative scenario for invalid restoredir set to "" we need to cleanup the member folder in the working directory
 	restoreDir := path.Clean("")
-	info, err := os.Stat(fmt.Sprintf("%s/%s", restoreDir, "member"))
+	// info, err := os.Stat(fmt.Sprintf("%s/%s", restoreDir, "member"))
+	info, err := os.Stat(path.Join(restoreDir, "member"))
```

Comment:
You don't need `os.Stat()` itself. You can directly run `os.RemoveAll(path.Join(restoreDir, "member"))`.